umbraco-cloud/set-up/manage-environments.md

```diff
@@ -1,35 +1,43 @@
 # Manage Environments
 
-When working with an Umbraco Cloud project, you can add or remove extra environments depending on the plan you are in:
+When working with an Umbraco Cloud project, you can add or remove extra environments depending on your plan:
 
-* For the Starter plan, you can add a Development environment for an additional price per month.
-* For the Standard plan, you get the Development environment for free and can add a Staging environment for an additional price per month.
-* For the Professional plan, you get the Development and Staging environment for free. Additionally, you can add and remove environments whenever you like without any additional cost.
+* **Starter plan:** Add a left-most mainline environment for an additional price per month.
+* **Standard plan:** The left-most mainline environment is included for free. Additional environments can be added for an additional price per month.
+* **Professional plan:** Mainline environments are included for free. You can add or remove environments at no extra cost.
```

Comment:
I'd recommend that you copy paste the table that I've also added in several other articles, like Project Overview and Flexible Environments.
It's a little easier to read, and we ensure get the same information in all articles.
I'll look into adding is a a reusable block later on 😅